addons/l10n_din5008/report/din5008_report.xml

```diff
@@ -255,6 +255,39 @@
                     </span>
                 </t>
             </xpath>
+            <!-- Insert Position Column as the First Column in the Table Header -->
+            <xpath expr="//table[@name='invoice_line_table']/thead/tr/th[1]" position="before">
+                <th name="th_position" class="text-start">Position</th>
+            </xpath>
+            <xpath expr="//table[@name='invoice_line_table']/tbody//tr[1]//td[1]" position="before">
+                <td class="text-start align-top"><t t-out="line_index+1"/></td>
+            </xpath>
+
+            <!--adding currency to unit price-->
+            <xpath expr="//td[@name='td_price_unit']/span" position="attributes">
+                <attribute name="t-options"  >{"widget": "monetary", "display_currency": o.currency_id}</attribute>
+            </xpath>
+
+            <!-- Align Quantity, Unit Price, Taxes, Discount, Amount to the top -->
+            <xpath expr="//td[@name='td_quantity']" position="attributes">
+                <attribute name="class" add="align-top" separator=" " />
+            </xpath>
+
+            <xpath expr="//td[@name='td_price_unit']" position="attributes">
+                <attribute name="t-attf-class" add="align-top" separator=" " />
+            </xpath>
+
+            <xpath expr="//td[@name='td_taxes']" position="attributes">
+                <attribute name="t-attf-class" add="align-top" separator=" " />
+            </xpath>
+
+            <xpath expr="//td[@name='td_subtotal']" position="attributes">
+                <attribute name="class" add="align-top" separator=" " />
+            </xpath>
```

Comment:
wondering if all those align top should not be put in the standard report_invoice, is it happening in the basic one ?

Reply:
It was requested only for the din 5008 
So I think that it is intended to be not align-top in the basic one.

Reply:
Yeah but maybe it's something that could be done in the basic template if the same issues happens, could you test and if it's just a problem with the din5008 (which i doubt, we can do it only here)

Reply:
Why ? maybe there is a class on top that make the thing work and then we need the same for din5008 :smile:

Reply:
a class on top of that ? I will check that